docs-source/content/userguide/managing-domains/model-in-image/overview.md

```diff
@@ -50,10 +51,30 @@ When you deploy a Model in Image domain resource:
 
 Model updates can be applied at runtime by changing the image, secrets, or WDT model config map after initial deployment. If the image name changes, or the domain resource `restartVersion` changes, then this will cause the introspector to rerun and generate a new domain home, and subsequently the changed domain home will be propagated to the domain's WebLogic pods using a rolling upgrade (each pod restarting one at a time). See [Runtime updates]({{< relref "/userguide/managing-domains/model-in-image/runtime-updates.md" >}}).
 
+#### Continuous integration and delivery 'CI/CD'
```

Comment:
I don't think this sections adds anything other than the x-ref to CI/CD, which I think is important but should provide some hint of how MII takes advantage of CI/CD. The content in the "Always use external state" section already exists in CI/CD/Copy domains.

Reply:
I don't think it hurts to repeat this stuff.  The idea is to call out important information for MII readers directly in the MII overview.

Reply:
@rjeberhard or @markxnelson Can you weigh in here?

Reply:
yes, i will get to it...

Reply:
@tbarnes-us The repeated section is not that big of a problem. Perhaps, the more important point to be made here is the lack of explanation for the x-ref to CI/CD. It doesn't need much, maybe just something like, "To understand how Model in Image works with CI/CD, see..." If users only wanted a "For a full discussion of CI/CD with the operator," they could have selected it from the menu.

Reply:
@rosemarymarano Does this look right now?  Please resolve if so.